tools/nl/embeddings/finetuning/README.md

```diff
@@ -1,24 +1,32 @@
 # Fine Tuning the Model
 
 A good guide to fine tuning Sentence Transformer models is [here](https://huggingface.co/blog/how-to-train-sentence-transformers).
-To fine tune starting from an existing finetuned model (based on sentence alternatives) using sentence pairs provided in [`finetuning/sentence_pairs.csv`](../data/finetuning/sentence_pairs.csv), do the following (this is the default behavior):
+
+There are two steps involved in a fully fine tuned model.
+
+1. **Intermediate Tuning**. This step starts with a base (pretrained) Sentence Transformer model, e.g. `all-MiniLM-L6-v2` and uses the human-curated and auto-generated alternative sentences/texts prepared for the embeddings indices. We use the `medium` index for which the alternatives are under [`data/alternatives/`](../data/alternatives/).
+
+2. **Final Tuning**. This steps uses an **Intermediate** finetuned model, i.e. the output from Step (1) above, and further finetunes the model use the manually provided sentence pairs (and associated scores) in [`finetuning/sentence_pairs.csv`](../data/finetuning/sentence_pairs.csv). 
+
+To fine tune starting from an existing **Intermediate** finetuned model, do the following (this should be much quicker than performing **both** the Intermediate and **Final** tuning):
 
 ```bash
-    ./run.sh
+    ./run.sh -f <tuned_intermediate_model_path_on_gcs>
```

Comment:
nit: can do `[-f <tuned_intermediate_model_path_on_gcs>]` to note it is optional?

Reply:
done